.github/workflows/deploy-integ.yml

```diff
@@ -48,6 +48,33 @@ jobs:
         run: |
           cp ./main/end-to-end-tests/e2eGitHubConfig.yml ./main/config/settings/${STAGE_NAME}.yml
           ./scripts/environment-deploy.sh ${STAGE_NAME}
+  integration-test:
+    name: Integration test
+    runs-on: ubuntu-18.04
+    needs: deploy
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Use Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: 12
+      - name: Install pnpm and system libraries
+        run: |
+          npm install -g pnpm
+          sudo apt-get install libgtk2.0-0 libgtk-3-0 libgbm-dev libnotify-dev libgconf-2-4 libnss3 libxss1 libasound2 libxtst6 xauth xvfb
+      - name: Configure AWS credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          STAGE_NAME: e2etest
+          AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: us-east-1
+      - name: Run integration tests
+        run: ./scripts/run-integration-tests.sh ${STAGE_NAME}
+        working-directory: ./
+        env:
```

Comment:
Do we need these vars as well?
INT_TEST_DIR
ENV_NAME

https://github.com/SanketD92/service-workbench-on-aws/blob/feat_GH_integ_workflow/scripts/run-integration-tests.sh

Reply:
ENV_NAME is the argument passed to this script, whereas INT_TEST_DIR _should_ be initialized as part of sourcing util.sh script (line 8 in run-integration-tests.sh)